front/pages/api/w/[wId]/assistant/agent_configurations/index.ts

```diff
@@ -295,6 +301,49 @@ export async function createOrUpgradeAgentConfiguration(
     if (agentConfigurationRes.value.status === "active") {
       trackAssistantCreated(auth, { assistant: agentConfigurationRes.value });
     }
+    if (actionConfig) {
+      // TODO(@fontanierh) Temporary to remove.
```

Comment:
Let's add a bit more context here :)

This is a shadow write while we invert the relationship between configuration and actions etc...

Reply:
sg